foundationsync/indico_foundationsync.py

```diff
@@ -119,7 +127,7 @@ def _parse_room_data(self, raw_data, coordinates, room_id):
             data['latitude'] = building_coordinates['latitude']
             data['longitude'] = building_coordinates['longitude']
 
-        return data, email_warning
+        return data, is_reservable, email_warning
```

Comment:
why not just put the correct `protection_mode` in the `data` in this function? then you don't need extra code to set it later since we copy all the data to the room in `_update_room`